### PR TITLE
Expand & fix exif-orientation test

### PR DESCRIPTION
**Pull request by lexaknyazev**
- The `imageOrientation` option is a string-based enum, not a boolean; it's now supported everywhere.
- Added tests for uploading `ImageBitmap` objects created from `Image` elements.

Passes in Firefox, various failures in WebKit and Chromium.

### CODE REVIEW

File: sdk/tests/conformance/textures/misc/exif-orientation.html [hunk 1]
```diff
@@ -56,24 +56,9 @@
...
-        try {
-            bitmap = await createImageBitmap(blob, {imageOrientation: flipY});
-        } catch (e) {
-            output("  (createImageBitmap options not supported - skipping flipY=true case)");
-            return;
-        }
-    } else {
-        bitmap = await createImageBitmap(blob);
-    }
+    const bitmap = await createImageBitmap(source, {imageOrientation: flipY ? 'flipY' : 'none'});
```
Comment (kkinnunen-apple) [inline sdk/tests/conformance/textures/misc/exif-orientation.html:61]: you could also do
`flipY ? {imageOrientation: flipY } : undefined`

File: sdk/tests/conformance/textures/misc/exif-orientation.html [hunk 2]
```diff
@@ -93,8 +78,14 @@
     let blob = await response.blob();
     output("----------------------------------------------------------------");
     output("Testing " + filename + " via ImageBitmap from Blob");
-    await testImageBitmapFromBlobWithFlipY(blob, true);
-    await testImageBitmapFromBlobWithFlipY(blob, false);
+    await testImageBitmapWithFlipY(blob, true);
```
Comment (kkinnunen-apple) [inline sdk/tests/conformance/textures/misc/exif-orientation.html:81]: in these you would do
await testImageBitmapWithFlipY(blob, undefined);
await testImageBitmapWithFlipY(blob, "flipY");
await testImageBitmapWithFlipY(blob, "none");

Comment (lexaknyazev) [reply sdk/tests/conformance/textures/misc/exif-orientation.html]: Done.

**Review by kenrussell — APPROVED** [review]
Apologies for the long delay approving this. I'd hoped to fix the underlying bug in Chromium before strengthening this test but that hasn't been possible.

**Pull request merged**